Common/FeatureFlags.swift

```diff
@@ -11,32 +11,73 @@ import Foundation
 let FeatureFlags = FeatureFlagConfiguration()
 
 struct FeatureFlagConfiguration: Decodable {
-    let sensitivityOverridesEnabled: Bool
```

Comment:
[nit] Reordering alphabetically? Note that @novalegra also has a pending PR with changes to this file that I think she will complete before your changes go in. You'll need to merge those changes in here as well.

Reply:
Yup. 

ok will do.

Reply:
They're merged in!